rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandler.groovy

```diff
@@ -132,6 +132,11 @@ public class AWSBakeHandler extends CloudProviderBakeHandler {
   }
 
   @Override
+  String getBaseCommand() {
+    return awsBakeryDefaults.templateFile == "aws-chroot.json" ? "sudo" : ""
```

Comment:
I think you also want to add an entry for `aws-chroot.json` just after the ebs builder here: https://github.com/spinnaker/rosco/blob/master/rosco-web/rosco-web.gradle#L94

Will make it easier for users to safely upgrade the package.